_posts/en/newsletters/2019-12-04-newsletter.md

```diff
@@ -0,0 +1,155 @@
+---
+title: 'Bitcoin Optech Newsletter #75'
+permalink: /en/newsletters/2019/12/04/
+name: 2019-12-04-newsletter
+slug: 2019-12-04-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes some recent discussion about the
+schnorr and taproot proposals, notes the recent name change of the
```

Comment:
I thought the name change was the least interesting part of your summary below. (Although I know you’re happy with the name change)

Reply:
Also thought the proposed BOLT News item was worthy of the summary